manifests/unbound_exporter.pp

```diff
@@ -0,0 +1,122 @@
+#
+# @summary This module manages prometheus unbound exporter. The exporter needs to be compiled by hand! (https://github.com/kumina/unbound_exporter/issues/21)
+#
+# @param arch
+#  Architecture
+# @param bin_dir
+#  Directory where binaries are located
+# @param download_extension
+#  Extension for the release binary archive
+# @param download_url
+#  Complete URL corresponding to the where the release binary archive can be downloaded
+# @param download_url_base
+#  Base URL for the binary archive
+# @param extra_groups
+#  Extra groups to add the binary user to
+# @param extra_options
+#  Extra options added to the startup command
+# @param group
+#  Group under which the binary is running
+# @param init_style
+#  Service startup scripts style (e.g. rc, upstart or systemd)
+# @param install_method
+#  Installation method: url or package (only url is supported currently)
```

Comment:
Is `url` supported?  Or it is if URL is to a local mirror where you've built the binary??
Maybe include an `@example Installing from your own package repository` or something based on how you're currently using this?

Reply:
at the moment I compiled the binary by myself and copied it into /usr/local/bin/unbound_exporter. I was hoping that https://github.com/kumina/unbound_exporter/issues/21 will be solved soon. That would enable this class to download the exporter from there. maybe we should keep this PR open until they provide proper releases.